### ISSUE TITLE
Keyboard navigation fail when there is disabled tab items

### ISSUE THREAD

**Issue by guanpu** [alibaba-fusion/next#145]
- [ ] I have searched the [issues](https://github.com/alibaba-fusion/next/issues) of this repository and believe that this is not a duplicate.

### Version
1.11.7

### Component
Tab

### Environment
ANY

### Reproduction link
[https://fusion.design/component/tab](https://fusion.design/component/tab)

### Steps to reproduce
Find the "disabled" demo at https://fusion.design/component/tab, click one and use left or right key to switch tab, it won't work

**Comment by guanpu** [comment]
It actually turn out to be not related with disable, just the demo page for disable didn't set keys properly.
